src/pool-cl/lens/TickLens.sol

```diff
@@ -27,22 +27,20 @@ contract TickLens is ITickLens {
         override
         returns (PopulatedTick[] memory populatedTicks)
     {
-        PoolId id = key.toId();
-        int24 tickSpacing = key.parameters.getTickSpacing();
-        return getPopulatedTicksInWord(id, tickSpacing, tickBitmapIndex);
+        return getPopulatedTicksInWord(key.toId(), tickBitmapIndex);
     }
```

Comment:
Also need to remove "import {TickMath} from "pancake-v4-core/src/pool-cl/libraries/TickMath.sol";"